data-model/fdo-type/create-update-tombstone-event/0.3.0/examples/update-event-example.json

```diff
@@ -45,12 +58,23 @@
       }
     },
     "prov:wasGeneratedBy": "7ba628d4-2e28-4ce4-ad1e-e99c97c20507",
-    "prov:wasRevisionOf": "https://hdl.handle.net/20.5000.1025/9FS-W6R-L1L/1",
-    "prov:wasAttributedTo": "https://hdl.handle.net/20.5000.1025/ASC-WQW-AAU/2"
+    "prov:wasRevisionOf": "https://hdl.handle.net/20.5000.1025/9FS-W6R-L1L/1"
   },
-  "prov:Agent": {
-    "@id": "https://hdl.handle.net/20.5000.1025/ASC-WQW-AAU/2",
-    "@type": "prov:SoftwareAgent",
-    "foaf:name": "orchestration-backend"
-  }
+  "ods:hasProvAgent": [
+    {
```

Comment:
Why does this not point towards the agent object?

Reply:
It does, this is a valid object. The agent-object is just a fixed set of properties we use for all places where we refer to agents.

Reply:
My bad its an example :0